index.html

```diff
@@ -447,14 +447,89 @@ <h3>Event Model</h3>
                         </ul>
                     </section>
                 </section>
+                <section>
+                  <section>
+                    <h3>Adaptive experiments</h3>
+                  </section>
+                  <section>
+                    <p>Feedback Paths</p>
+                    <ul>
+                      <li> data quality checks </li>
+                      <li> prompt / real-time analysis to steer experiment
+                        <ul>
+                          <li> "human-in-the-loop" </li>
+                          <li> "computer-in-the-loop"  </li>
+                        </ul>
+                      </li>
+                    </ul>
+                  </section>
+                  <section>
+                    <p>Scales of Adaptive-ness</p>
+                    <ol class="arabic simple">
+                      <li>below <code class="literal notranslate"><span class="pre">bluesky</span></code> & <code class="literal notranslate"><span class="pre">ophyd</span></code> </li>
+                      <li>in <code class="literal notranslate"><span class="pre">bluesky</span></code> plans, but without generating <a class="reference external" href="https://blueskyproject.io/event-model/api.html#event_model.DocumentNames.event" title="(in Bluesky Event Model v1.15.2.post5+g842bdb8)"><code class="literal notranslate"><span class="pre">event</span></code></a></li>
+                      <li>providing feedback on a per-<a class="reference external" href="https://blueskyproject.io/event-model/api.html#event_model.DocumentNames.event" title="(in Bluesky Event Model v1.15.2.post5+g842bdb8)"><code class="literal notranslate"><span class="pre">event</span></code></a> basis</li>
+                      <li>providing feedback on a per-run / <a class="reference external" href="https://blueskyproject.io/event-model/api.html#event_model.DocumentNames.start" title="(in Bluesky Event Model v1.15.2.post5+g842bdb8)"><code class="literal notranslate"><span class="pre">start</span></code></a> basis</li>
+                      <li>providing feedback across many runs</li>
+                      <li>asynchronous and decoupled feedback</li>
+</ol>
+                  </section>
+                  <section>
+                    <p>below <code class="literal notranslate"><span class="pre">bluesky</span></code> & <code class="literal notranslate"><span class="pre">ophyd</span></code> </p>
+                    <ul>
+                      <li> timescale: ≫ 10Hz </li>
+                      <li> very limited time budget for analysis </li>
+                      <li> very limited access to data </li>
+                      <li> tightly coupled to hardware (PID loop, FGPA) </li>
+                      <li> expensive to write </li>
```

Comment:
FWIW, I initially misread "write" in the sense of writing data, being primed into that context by the points above. On a moment's thought it's clear, but maybe a word like "build" or "make" would by unambiguous.